backend/src/main/java/capstone/myfinancemanager/manager/exceptions/GlobalExceptionHandler.java

```diff
@@ -47,4 +47,13 @@ public ResponseEntity<Map<String, Object>> handlePasswordNotMatchException() {
 
         return new ResponseEntity<>(responseBody, HttpStatus.BAD_REQUEST);
     }
+
```

Comment:
Why is this commented out? Either delete it or uncomment. No dead code in main, please.